src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, int radius, String color) {
+        super(name, color);
+        this.radius = radius;
+    }
+
+    public String drawFigure() {
```

Comment:
```suggestion
    @Override
    public String drawFigure() {
```

let's add `@Override` annotation